index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <main id="main">
+
+      <h1 id="title">Dr. Norman Borlaug</h1>
+      <h3 id="sub-title">The man who saved a billion lives</h3>
+      
+      <div id="img-div">
+        <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt="" id="image">
```

Comment:
Kindly make sure that the comment has a suggestion/polite tone so that the student will not feel commanded. 😉 😊 
something like: I think it will be better to use...
or, kindly make use of...

Reply:
- Comment edited to reflect changes